src/main/java/entities/PublishedMaze.java

```diff
@@ -0,0 +1,10 @@
+package entities;
+
+import java.time.LocalDate;
+
+public class PublishedMaze extends Maze{
+    String author;
+    String name;
```

Comment:
make this id instead of name

Reply:
right after published maze is created, create an unique id for the published maze somehow.

one idea on how to generate the id is to concatenate the hash codes of the name of the author and the the local data object

Reply:
then within the data base, we can store the id's of the mazes created by each user

Reply:
Have name and id